src/_gettsim/parameters/elterngeld.yaml

```diff
@@ -172,3 +172,21 @@ max_monate_ind:
   2007-01-01:
     scalar: 12
     reference: Art. 1 G. v. 05.12.2006 BGBl. I S. 2748
+elterngeld_anr_frei:
```

Comment:
Cannot you just use the parameter `mindestbetrag`?

Reply:
Yes, I used the parameter which was in `grunds_im_alter` before but I guess the other solution is more elegant, I will change it